vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -47,32 +44,32 @@ packages:
          - OVMF
          - patch
          - psmisc
-         - python-pip
-         - python-requests
-         - python-setuptools
+         - python3-pip
+         - python3-requests
+         - python3-setuptools
```

Comment:
These packages won't exist on centos7 so we shouldn't change them, we probably need to deprecate centos7 suppport or figure out a way for this to temporarily work with python2 as well as python3 (on Centos/RHEL8 and Ubuntu)

Reply:
Oh I think this was addressed in the rebase commit, it may be worth squashing that into the original commit then force-pushing if you do any other updates